ts/Dashboard/Component/Component.ts

```diff
@@ -322,6 +324,16 @@ abstract class Component<TEventObject extends Component.EventTypes = Component.E
     }
 
     public setStore(store: Component.StoreTypes | undefined): this {
+        // Clean up old event listeners
+        if (this.tableEvents.length) {
+            while (this.tableEvents.length) {
```

Comment:
Assign the length to the variable, as a result, the code does not have to calculate the length of the array multiple times.